src/apolloContexters.ts

```diff
@@ -1,14 +1,52 @@
+/* eslint-disable */
+/* tslint:disable */
+// @ts-ignore
+
+
 import { Request, Response } from 'express'
+import DataLoader from 'dataloader'
+import { Container } from 'typedi'
+import { AdsRepository } from './repositories/AdsRepository'
+import AdsModel from './db/models/ads'
+
+Container.set('AdsModel', AdsModel)
+
+
+// Dependency injection with TypeDi
+const adsRepository = Container.get(AdsRepository)
+// @ts-ignore
+let loaderCounter = 0
+// @ts-ignore
+let tokenCounter = 0
+
+
+// Create DataLoaders batching and caching graphql requests
+function createAdLoader() {
+  const adLoader = new DataLoader(async (ids: readonly string[]) => {
+    console.log('Analyzing batch function 🔥🔥🔥🔥🔥🔥', ids)
+    if (!adsRepository) {
+      console.log('🤯🤯🤯🤯🤯🤯🤯🤯🤯🤯🤯 Could instantiate the adsRepository properly')
```

Comment:
Create a real Logger for this

Reply:
shut up stupid

Reply:
I do what I want cause I am super owner, your comment is not relevant keep working employee

Reply:
MTF